src/core/device.c

```diff
@@ -1064,7 +1064,6 @@ const UnitVTable device_vtable = {
                 "Device\0"
                 "Install\0",
 
-        .refuse_after = true,
```

Comment:
i don't get this commit. please explain. A device pops up whenever it likes, and we act on it and modify job states as soon as we take notice. we don't consult the ordering of the job queue anymore at that point.

Reply:
The point of this commit is to allow a device unit to have ordering constraints such as the ones written by generator_write_device_deps() and prevent PID1 from dropping them.

If a service is started and it requires a device and the device unit uses `After=foo.service`, a job for the device is queued and set in "waiting" state until foo.service is started. Then the device job is put in "running" state until the device pops up.

This allows to define 2 different types of job timeouts for devices, see commit a2df3ea4ae058693bc7bf203d144e8af3c9493d2 and 4195077ab4c823c385cdc3c868a56178f8d93723.

Reply:
@poettering does that make sense ?

Reply:
i see. 

can you please add this explanation to the commit msg? i.e. say explicitly that this is about having different timeouts.